packages/x-data-grid/src/models/gridBaseSlots.ts

```diff
@@ -284,11 +285,16 @@ export type SwitchProps = {
   checked?: boolean;
   onChange?: React.ChangeEventHandler;
   size?: 'small' | 'medium';
+  label?: React.ReactNode;
+  labelPlacement?: 'start' | 'end';
+  className?: string;
+  fullWidth?: boolean;
```

Comment:
I don't see `labelPlacement` and `fullWidth` being used, are they useful?

Reply:
No longer used, thanks for spotting.